src/bootstrap/src/core/config/config.rs

```diff
@@ -16,7 +16,6 @@ use std::{cmp, env, fs};
 use build_helper::exit;
 use build_helper::git::{GitConfig, get_closest_merge_commit, output_result};
 use serde::{Deserialize, Deserializer};
-use serde_derive::Deserialize;
```

Comment:
I see the rustbot mentioning the change to this file. I found the duplicate Deserialize error popped up out of nowhere and needed to remove it to unblock my build. Will revisit it to see if it's really necessary.

Reply:
seems like it's really an error but somehow it got flagged out in this PR. Maybe #69291?